webproject/src/main/java/com/google/sps/data/Room.java

```diff
@@ -54,23 +53,25 @@ private Room (Builder builder) {
 
     public double getDeliveryFee() {return deliveryFee;}
 
-    public List<String> getUsers() {return users;}
-
     public String getCreator() {return creator;}
 
     public double getOrdersValue() {return ordersValue;}
 
-    public void addUser(String userEmail) {
-        this.users.add(userEmail);
-    }
-
     public void addOrdersValue(double orderPrice) {
         this.ordersValue += orderPrice;
     }
 
+    public void addUser(String userEmail, String roomId) {
+        UserRoom userRoom = new UserRoom(userEmail, roomId);
+        userRoom.save();
+    }
+
     public void save() throws IOException {
         String roomJson = gson.toJson(this);
-        Firebase.sendRequest("https://summer20-sps-47.firebaseio.com/rooms.json", "POST", roomJson);
+        String firebaseResponse = Firebase.sendRequest("https://summer20-sps-47.firebaseio.com/rooms.json", "POST", roomJson);
+        Map<String, String> firebaseData = gson.fromJson(firebaseResponse, Map.class);
+
+        this.addUser(this.creator, firebaseData.get("name"));
```

Comment:
is name the unique identifier of a user?

Reply:
name is the key for the ID from firebase's response!